elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ElytronOidcSubsystemSchema.java

```diff
@@ -52,9 +93,10 @@ public enum ElytronOidcSubsystemSchema implements PersistentSubsystemSchema<Elyt
     VERSION_1_0(1, Stability.DEFAULT),
     VERSION_2_0(2, Stability.DEFAULT),
     VERSION_2_0_PREVIEW(2, 0, Stability.PREVIEW), // WildFly 32.0-present
+    VERSION_3_0_PREVIEW(3, 0, Stability.PREVIEW), // WildFly 33.0-present
     ;
 
-    static final Map<Stability, ElytronOidcSubsystemSchema> CURRENT = Feature.map(EnumSet.of(VERSION_2_0_PREVIEW, VERSION_2_0));
+    static final Map<Stability, ElytronOidcSubsystemSchema> CURRENT = Feature.map(EnumSet.of(VERSION_3_0_PREVIEW, VERSION_2_0_PREVIEW, VERSION_2_0));
```

Comment:
This is map of current schema per stability.  Calling Feature.map(...) with multiple values for a given stability is not recommended.  Instead use:
```java
Feature.map(EnumSet.of(VERSION_3_0_PREVIEW, VERSION_2_0));
```

Reply:
Fixed!